jekyll/_cci2/overview.md

```diff
@@ -0,0 +1,128 @@
+description: "High-level overview of the CircleCI Installation process."
+---
+This article provides an overview of CircleCI 2.0 installation, features, environments, and architecture in the following sections:
+
+* TOC
+{:toc}
+
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. CircleCI is installable inside your private cloud or data center and is free to try for a limited time. 
+
+CircleCI 2.0 provides new infrastructure that includes the following improvements:
+* New configuration with any number of jobs and workflows to orchestrate them. 
+* Custom images for execution on a per-job basis.
+* Fine-grained performance with custom caching and per-job CPU or memory allocation. 
+
+Refer to [Sign Up and Try CircleCI]({{ site.baseurl }}/2.0/first-steps/) to get started with the hosted application.
+
+## Trial Installation Requirements
+
+- A VM with at least 8GB of RAM, 100GB of disk space on the root volume, and a version of Linux that supports Docker, for example Ubuntu Trusty 14.04. Must have network connectivity to the GitHub or GitHub Enterprise instance and outbound internet access (proxy server is supported by request to CircleCI customer service).  
+
+OR
+
+- An Amazon EC2 VM instance with at least 8G of RAM, outbound internet access, and network connectivity to the GitHub or GitHub Enterprise instance.
+
+See the [CircleCI Trial Installation]({{ site.baseurl }}/enterprise/single-box/) document for trial installation instructions and links to the machine images.
+
+## Production Installation Requirements
+
+- Minimum of two instances, one for the Services component and one for the Builders. Both instances require running a Linux distribution with Docker support, such as Ubuntu Trusty, or the CircleCI Amazon Machine Image (AMI). 
+- The Services instance must have at least eight CPUs and 32GB RAM. 
+- The Builders instance must have six CPUs or GPUs and 10GB RAM and a root volume of at least 50GB.
+- Both instances must have network access to a GitHub or Github Enterprise instance. 
+
+To determine the size required for your workload, provision two CPUs for processing plus two CPUs and 4GB of memory for each container. For example, six CPUs and 10GB of RAM supports two containers concurrently because the default container size is two CPUs with 4GB. 
+
+### Production Installation Options
+
+It is possible to install CircleCI 2.0 in either of the following configurations:
+
+- A single Services machine with one or more Builder machines. See [Installing CircleCI on Amazon Web Services Manually]({{ site.baseurl }}/enterprise/aws-manual/) or [Installing CircleCI on Amazon Web Services with Terraform]({{ site.baseurl }}/enterprise/aws/) for instructions.
+- A single Services machine with four database hosts for HA and one or more Builder machines, see [Installing CircleCI in a High Availability Configuration]({{ site.baseurl }}/enterprise/high-availability/) for instructions.
+
+## Customer Use Cases 
```

Comment:
Is there any indication these have driven traffic? They feel out of place in docs, IMO.